src/main/java/com/denizenscript/ddiscordbot/objects/DiscordGroupTag.java

```diff
@@ -375,6 +378,26 @@ public static void registerTags() {
             }
             return new ElementTag(bestMatch.getId());
         });
+
+        // <--[tag]
+        // @attribute <DiscordGroupTag.users_with_roles[<role>|...]>
+        // @returns ListTag(DiscordUserTag)
+        // @plugin dDiscordBot
+        // @description
+        // Returns a list of all users in the group who have all the specified roles.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "users_with_roles", (attribute, object) -> {
+            if (!attribute.hasParam()) {
+                attribute.echoError("Invalid group.users_with_roles[...] tag: must have an input value.");

```

Comment:
This error is redundant in modern Denizen, it's tracked by the system

Reply:
Removed